style.css

```diff
@@ -0,0 +1,116 @@
+#nav_header {
+  display: flex;
+  align-items: center;
+  width: 375px;
+  height: 40px;
+  margin: auto;
+  padding: 0 px;
+  margin-top: 5vw;
+}
+
+.cellular_connection {
+  margin-left: 10px;
+}
+
+.wifi {
+  margin-left: 10px;
+}
+
+.battery {
+  margin-left: 10px;
+}
+
+.time > p {
+  display: inline;
+  font-family: 'Roboto', sans-serif;
+}
+
+.space {
+  height: 10px;
+  width: 10px;
+  flex: 6;
+}
+
+nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 375px;
+  margin: auto;
+  margin-top: 10px;
+}
+
+.nav_logo > h1 {
+  font-family: 'Poppins', sans-serif;
+  color: #6070ff;
+  font-size: 1.5em;
+  display: inline;
+}
+
+.navMenu {
+  font-family: sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.background {
+  height: 10px;
+  width: 375px;
+  margin: auto;
+}
+
+#content {
+  width: 375px;
+  height: 10px;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
```

Comment:
- [ ] **[OPTIONAL]** Kindly we could delete these icons 👇🏽 they are for reference that it is a mobile version.

![image](https://user-images.githubusercontent.com/58642949/186790931-ccb2cba5-61b5-4f22-bfbd-520c54b2558a.png)


- [ ] Please 🙏🏽 correct the margins of your design to be similar to the Figma template:

- Your design

![image](https://user-images.githubusercontent.com/58642949/186790734-3d1c1c8d-0491-4a30-92f5-5df5142f7005.png)

- Figma template

![image](https://user-images.githubusercontent.com/58642949/186790767-d0561fdf-2d3f-4c04-9e8f-416f96afd5a3.png)